spec/requests/api/v1/docs/readinglist_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "rails_helper"
```

Comment:
I think I covered most of it but wasn't super certain as it's the first time I'm writing rswag docs for the api.